.github/workflows/build-beta.yml

```diff
@@ -25,48 +25,28 @@ jobs:
           # Specifying `ref` ensures that the head commit is checked out directly.
           ref: ${{ env.HEAD_COMMIT_HASH }}
 
-      - name: Needs beta build
-        if: ${{ env.BRANCH != 'master' }}
-        id: needs-beta-build
-        run: |
-          version="${BRANCH/Version-v/}"
-          commit_message=$(git show -s --format=%s HEAD)
-          beta_version_regex="Version v[0-9]+\.[0-9]+\.[0-9]+-beta\.[0-9]+"
-
-          if [[ "$commit_message" =~ $beta_version_regex ]]; then
-              printf '%s\n' "Creating a build for $version with $commit_message"
-              echo "NEEDS_BETA_BUILD=true" >> "$GITHUB_OUTPUT"
-          else
-              printf '%s\n' 'Commit message does not match commit message for beta pattern; skipping beta build'
-              echo "NEEDS_BETA_BUILD=false" >> "$GITHUB_OUTPUT"
-          fi
-
       - name: Checkout and setup high risk environment
-        if: ${{ steps.needs-beta-build.outputs.NEEDS_BETA_BUILD == 'true' }}
         uses: MetaMask/action-checkout-and-setup@v1
         with:
           is-high-risk-environment: true
           ref: ${{ env.HEAD_COMMIT_HASH }}
           skip-allow-scripts: true
 
       - name: Run beta build
-        if: ${{ steps.needs-beta-build.outputs.NEEDS_BETA_BUILD == 'true' }}
         env:
-          INFURA_PROJECT_ID: ${{ vars.INFURA_PROJECT_ID }}
-          INFURA_BETA_PROJECT_ID: ${{ vars.INFURA_BETA_PROJECT_ID }}
+          INFURA_PROJECT_ID: ''
+          INFURA_BETA_PROJECT_ID: ''
           SEGMENT_BETA_WRITE_KEY: 00000000000
           SENTRY_DSN: https://fake@sentry.io/0000000
```

Comment:
We don't need a fake DSN here, we have a fake fallback built-in to the application already. It looks like this was added to get around a validation error for the production build, which wasn't supposed to be called here in the first place (and which of course should be passed the _real_ DSN, not a fake one).

However, we do have a separate secret for non-production builds that is unset here. Sentry is effectively disabled for beta builds right now, but it should be enabled and routing errors to our test project. To do that, we'd want to set the environment `SENTRY_DSN_DEV` here to our development DSN. I've just added that to secrets as `SENTRY_DSN_DEV`.

Reply:
Done.